api/pom.xml

```diff
@@ -31,9 +31,39 @@
             <archive>users@mvc-spec.java.net</archive>
         </mailingList>
     </mailingLists>
+
+    <properties>
+        <api_package>javax.mvc</api_package>
+        <spec_version>1.0</spec_version>
+        <spec_impl_version>1.0-SNAPSHOT</spec_impl_version>
```

Comment:
Could we use `${project.version}` here? Not sure if using properties inside `<properties>` works, but having the version at two places doesn't feel right.